cvat-ui/src/components/annotation-page/standard-workspace/canvas-wrapper.tsx

```diff
@@ -478,8 +478,10 @@ export default class CanvasWrapperComponent extends React.PureComponent<Props> {
         const {
             state,
             points,
+            annotation_type,
         } = event.detail;
         state.points = points;
+        state.annotation_type = annotation_type;
```

Comment:
I expect it is always `Manual`. Right?

Reply:
Yes, when drawn by the user it is always `Manual`

Reply:
> Yes, when drawn by the user it is always `Manual`

Then we don't need to pass ``annotation_type`` value from ``cvat-canvas``. Just to assign const ``Manual`` here

Reply:
As suggested, I passed the value in `canvas-wrapper.tsx`  onCanvasShapeDrawn and onCanvasShapeEditDone instead of passing the value through `cvat-canvas`. The field does become `undefined` in case of propagate. I'm not sure I understand how  propagate works. I need a little help here.

Reply:
@Priya4607 

Propagate feature isn't related anyhow with canvas.
Look at ``propagateObjectAsync()`` in ``cvat-ui/src/actions/annotation-actions.ts``
Probably you just do not copy the field